### PR TITLE
Bump to JDK 11 and replace jenkins jira api with atlassian one

### PR DESCRIPTION
**Pull request by NotMyFault**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -64,24 +64,43 @@
...
+        <exclusion>
+          <groupId>org.apache.httpcomponents</groupId>
+          <artifactId>httpcore</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
+    <dependency>
+      <groupId>io.atlassian.util.concurrent</groupId>
+      <artifactId>atlassian-util-concurrent</artifactId>
+      <version>4.0.1</version>
```
Comment (NotMyFault) [inline pom.xml:92]: Overwriting pircbotx' dependencies, blocked by https://github.com/pircbotx/pircbotx/issues/411

File: pom.xml [hunk 2]
```diff
@@ -101,17 +120,12 @@
     <dependency>
       <groupId>org.kohsuke</groupId>
       <artifactId>github-api</artifactId>
-      <version>1.112</version>
-    </dependency>
-    <dependency>
-      <groupId>org.javatuples</groupId>
```
Comment (slide) [inline pom.xml:107]: Is this even used still now that the hosting stuff has been removed?

**Review by slide — APPROVED** [review]

**Comment by lemeurherve** [comment]
@slide FYI I've added you with "write" permission to this repository.

File: pom.xml (hunk 2 above)
Comment (slide) [reply pom.xml:107]: I submitted #127 to clean this up.

**Review by timja — APPROVED** [review]

**Comment by NotMyFault** [comment]
> @slide FYI I've added you with "write" permission to this repository to be able to approve PRs.

A `hacktoberfest` badge would suit this repository pretty well :P

**Pull request merged**